Apps/Admin/Server/Services/SupportService.cs

```diff
@@ -114,57 +104,43 @@ public class SupportService : ISupportService
             PatientModel patient;
 
             if (query.QueryType == ClientRegistryType.Hdid)
             {
-                patient = await this.GetPatientAsync(new(Hdid: query.QueryParameter, Source: PatientDetailSource.All, UseCache: false), ct).ConfigureAwait(true);
+                patient = await this.GetPatientAsync(new(Hdid: query.QueryParameter, Source: PatientDetailSource.All, UseCache: false), ct);
             }
             else
             {
-                patient = await this.GetPatientAsync(new(query.QueryParameter, Source: PatientDetailSource.Empi, UseCache: false), ct).ConfigureAwait(true);
+                patient = await this.GetPatientAsync(new(query.QueryParameter, Source: PatientDetailSource.Empi, UseCache: false), ct);
             }
 
-            if (query.IncludeMessagingVerifications)
-            {
-                messagingVerifications = await this.GetMessagingVerificationsAsync(patient.Hdid, ct);
-            }
+            Task<VaccineDetails>? getVaccineDetails =
+                query.IncludeCovidDetails ? this.GetVaccineDetails(patient, query.RefreshVaccineDetails) : null;
 
-            if (query.IncludeBlockedDataSources)
-            {
-                blockedDataSources = await this.GetBlockedDataSourcesAsync(patient.Hdid, ct);
-            }
+            Task<CovidAssessmentDetailsResponse>? getCovidAssessmentDetails =
+                query.IncludeCovidDetails ? this.immunizationAdminApi.GetCovidAssessmentDetails(new() { Phn = patient.Phn }, this.GetAccessToken()) : null;
 
-            if (query.IncludeAgentActions)
-            {
-                agentActions = await this.GetAgentActionsAsync(patient.Hdid, ct);
-            }
+            IEnumerable<MessagingVerificationModel>? messagingVerifications =
+                query.IncludeMessagingVerifications ? await this.GetMessagingVerificationsAsync(patient.Hdid, ct) : null;
 
-            if (query.IncludeDependents)
-            {
-                dependents = await this.GetDependentsAsync(patient.Hdid, ct);
-            }
+            IEnumerable<DataSource>? blockedDataSources =
+                query.IncludeBlockedDataSources ? await this.GetBlockedDataSourcesAsync(patient.Hdid, ct) : null;
 
-            if (query.IncludeCovidDetails)
-            {
-                vaccineDetails = await this.GetVaccineDetails(patient, query.RefreshVaccineDetails).ConfigureAwait(true);
-                covidAssessmentDetails = await this.immunizationAdminApi.GetCovidAssessmentDetails(new() { Phn = patient.Phn }, this.GetAccessToken()).ConfigureAwait(true);
-            }
+            IEnumerable<AgentAction>? agentActions =
+                query.IncludeAgentActions ? await this.GetAgentActionsAsync(patient.Hdid, ct) : null;
+
+            IEnumerable<PatientSupportDependentInfo>? dependents =
+                query.IncludeDependents ? await this.GetAllDependentInfoAsync(patient.Hdid, ct) : null;
 
             PatientSupportDetails details = new()
```

Comment:
No Change:

I'm surprised the linter didn't suggest this to be an immediate return:

```CSharp
return new () 
{
  // ... all properties here
}
```